src/Lykke.Job.CandlesHistoryWriter.Core/Lykke.Job.CandlesHistoryWriter.Core.csproj

```diff
@@ -8,7 +8,7 @@
   <ItemGroup>
     <PackageReference Include="Autofac" Version="4.6.2" />
```

Comment:
I almost sure that Autofac reference can be deleted as it comes transiently via other own libs